src/main/java/com/gooddata/http/client/GoodDataHttpClient.java

```diff
@@ -153,12 +169,24 @@ public GoodDataHttpClient(final SSTRetrievalStrategy sstStrategy) {
      * @param sstStrategy super-secure token (SST) obtaining strategy
      */
     public GoodDataHttpClient(final HttpClient httpClient, final HttpHost authHost, final SSTRetrievalStrategy sstStrategy) {
+        this(httpClient, authHost, sstStrategy, null);
+    }
+
+    /**
+     * Construct object.
+     * @param httpClient Http client
+     * @param authHost http host
+     * @param sstStrategy super-secure token (SST) obtaining strategy
+     * @param gdcHostHeader X-GDC_HOST header value
```

Comment:
Please unify naming and explanation in javadoc with one in gooddata-java (https://github.com/martiner/gooddata-java/pull/317)